lib/foreman_maintain/cli/packages_command.rb

```diff
@@ -51,7 +51,7 @@ def execute
       end
 
       subcommand 'is-locked', 'Check if update of packages is allowed' do
-        interactive_option
+        interactive_option([])
```

Comment:
The installer uses this:
https://github.com/theforeman/foreman-installer/blob/bc575c2d662dad4779623b89d24ca7750dcec3bd/hooks/pre_commit/09-version_locking.rb#L2

I'd say we should patch the installer (https://github.com/theforeman/foreman-installer/pull/701) but considering downstream shipping perhaps this should also accept `assumeyes`?

Reply:
for the record - [the issue](https://projects.theforeman.org/issues/30094) linked to this PR also expects the `assumeyes` to be available for this subcommand (see "expected result")

Reply:
@ekohl technically `assumeyes` is useless for `is-locked` command as there isn't any user interaction happening in the process. But if we drop the `--assumeyes` from the foremain-installer then it will just increase the one more `if else` block in maintain because there will be older versions of `foreman-installer` which are still using the `--assumeyes`. Hence I propose not to remove `--assumeyes` from the installer, and I readd it for the `is-locked`.

Reply:
Isn't `--assumeyes` also useless in older versions? So the installer could safely drop it without creating an `else` clause? The behavior should be the same, right?

Reply:
Yes its useless for older versions. If you can cherry pick the installer changes in older versions too then we can keep this as is and change will only happen to installer.